src/query/query_executor.c

```diff
@@ -24326,6 +24326,38 @@ qexec_execute_build_columns (THREAD_ENTRY * thread_p, XASL_NODE * xasl, XASL_STA
 	      db_make_string (out_values[idx_val], "auto_increment");
 	    }
 
+    /* attribute is invisible or not */
+    if (attrepr->is_invisible)
+	    {
+	      const char *saved = db_get_string (out_values[idx_val]);
+	      size_t len = strlen (saved);
+
+	      /* add whitespace character if saved is not an empty string */
+	      const char *is_invisible_string = "INVISIBLE";
```

Comment:
```suggestion
	      const char *is_invisible_string = "invisible";
```

INVISIBLE 키워드를 소문자로 변경이 필요해 보입니다.

매뉴얼과 실제 테스트 결과를 보면, auto_increment는 show columns에서 소문자로 표기되지만 INVISIBLE은 현재 대문자로 출력되고 있습니다.
- 매뉴얼 :
https://www.cubrid.org/manual/ko/11.4/sql/query/show.html#show-columns

- 예 :
```sql
create table t1 (col1 int auto_increment, col2 int INVISIBLE);
insert into t1 values(1,1);

SHOW COLUMNS FROM t1;

  Field                 Type                  Null                  Key                   Default               Extra               
====================================================================================================================================
  'col1'                'INTEGER'             'YES'                 ''                    NULL                  'auto_increment'    
  'col2'                'INTEGER'             'YES'                 ''                    NULL                  'INVISIBLE'  
```